script/Common/00_00.txt

```diff
@@ -1878,7 +1883,7 @@ I was really scared before I met you.
 [sha:393eb95d76041b4279cfbaa3ca6172090b08db02]{
 -- Page 141, Offset 465.
 -- ……だめなんだ。先生がいなくちゃ、こんなメガネがあったってだめに決まってるじゃないか……！」
-...It's no good. What use are these glasses if I don't have you around...!"
+So, I can't... even with these glasses, I obviously can't do it without you...!"
```

Comment:
Kinda prefer orig line

Reply:
I feel like the old one is bordering on straight up wrong.

Reply:
Can you elaborate? I get the same meaning from both of these lines, the original just feels more fluent

Reply:
「―――無理だよ先生。僕だけじゃ良いことになんてできない
　ほんとは先生に会うまで恐くてたまらなかった。けど先生がいてくれたから、僕は僕に戻れたんだ。
　……だめなんだ。先生がいなくちゃ、こんなメガネがあったってだめに決まってるじゃないか……！」
　
　だめなんだ as in "he can't deal with this", not some vague "it's no good" (what isn't?). This reflects back on the start of this line. 僕だけじゃ良いことになんてできない
The things with the glasses is subtly off too because it sounds more like "he needs her around if the glasses are gonna be of any help", when its more like "just because I have these glasses doesn't mean I can handle this one my own!"

Reply:
Ah, yeah I guess you could read it that way. Still feels like the sentence as-is is a bit stiff to me, might suggest
> I just can't... Even if I have these glasses, there's no way I can do it without you around!